clientV2/assets/components/_modal.scss

```diff
@@ -0,0 +1,68 @@
+$primaryGray: #777777;
+
+.modal {
+  top: 0;
+  left: 0;
+  height: 100vh;
+  right: 0;
+  background-color: rgba(54, 93, 220, 0.3);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  z-index: 1040;
```

Comment:
I think this value is too much, you can use something like 2 or 3, for the sake of easy debugging